Fabric.Terminology.API/Logging/LogFactory.cs

```diff
@@ -1,25 +1,48 @@
 ﻿namespace Fabric.Terminology.API.Logging
 {
+    using Fabric.Terminology.API.Configuration;
+
     using Serilog;
     using Serilog.Core;
 
     /// <summary>
     /// Responsible for creating the <see cref="ILogger"/>
     /// </summary>
-    /// <remarks>
-    /// This class should be removed with Fabric.Platform logging has been finalized.
-    /// </remarks>
     internal class LogFactory
     {
-        public static ILogger CreateLogger(LoggingLevelSwitch levelSwitch)
+        public static ILogger CreateTraceLogger(LoggingLevelSwitch levelSwitch, ApplicationInsightsSettings appInsightsConfig)
+        {
+            var loggerConfiguration = CreateLoggerConfiguration(levelSwitch);
+
+            if (appInsightsConfig != null && appInsightsConfig.Enabled &&
+                !string.IsNullOrEmpty(appInsightsConfig.InstrumentationKey))
+            {
+                loggerConfiguration.WriteTo.ApplicationInsightsTraces(appInsightsConfig.InstrumentationKey);
+            }
+
+            return loggerConfiguration.CreateLogger();
+        }
+
+        public static ILogger CreateEventLogger(LoggingLevelSwitch levelSwitch, ApplicationInsightsSettings appInsightsConfig)
+        {
+            var loggerConfiguration = CreateLoggerConfiguration(levelSwitch);
+
+            if (appInsightsConfig != null && appInsightsConfig.Enabled &&
+                !string.IsNullOrEmpty(appInsightsConfig.InstrumentationKey))
+            {
+                loggerConfiguration.WriteTo.ApplicationInsightsEvents(appInsightsConfig.InstrumentationKey);
+            }
+
+            return loggerConfiguration.CreateLogger();
+        }
+
+        private static LoggerConfiguration CreateLoggerConfiguration(LoggingLevelSwitch levelSwitch)
         {
-            // TODO work on log format
             return new LoggerConfiguration()
                 .MinimumLevel.ControlledBy(levelSwitch)
                 .Enrich.FromLogContext()
-                .WriteTo.LiterateConsole()
-                .WriteTo.RollingFile("Logs\\fabric-terminology-{Date}.txt")
-                .CreateLogger();
+                .WriteTo.ColoredConsole()
+                .WriteTo.RollingFile("Logs\\fabric-terminology-{Date}.txt");
```

Comment:
You may want to do a `Path.Join` here as it will handle if you are on a unix or windows box